Description: Do not package m2s-clcc or m2s-glcc
 m2s-clcc and m2s-glcc depend on the AMDAPP SDK which is currently not
 packaged. Do not bother including these binaries in any package until
 AMDAPP SDK has been packaged.
 .
 multi2sim (4.1+svn20130603-1) unstable; urgency=low
 .
   * Do not package m2s-clcc or m2s-glcc
Author: William Panlener <wpanlener@gmail.com>

---

--- multi2sim-4.1+svn20130603.orig/configure.ac
+++ multi2sim-4.1+svn20130603/configure.ac
@@ -861,16 +861,6 @@ AC_CONFIG_FILES([
 		src/lib/esim/Makefile
 		src/lib/mhandle/Makefile
 		src/lib/util/Makefile
-
-		tools/Makefile
-
-		tools/clcc/Makefile
-		tools/clcc/amd/Makefile
-		tools/clcc/cl2llvm/Makefile
-		tools/clcc/llvm2si/Makefile
-		tools/clcc/si2bin/Makefile
-
-		tools/glcc/Makefile
 		])
 AC_OUTPUT
 
--- multi2sim-4.1+svn20130603.orig/Makefile.am
+++ multi2sim-4.1+svn20130603/Makefile.am
@@ -1,7 +1,6 @@
 SUBDIRS = \
 	runtime \
-	src \
-	tools
+	src
 
 images = \
 	images/alu_engine.png \
